nuget/framework/nunit.nuspec

```diff
@@ -28,8 +28,9 @@ Supported platforms:
       <group targetFramework="net35" />
       <group targetFramework="net40" />
       <group targetFramework="net45" />
-      <group targetFramework="netstandard1.6">
+      <group targetFramework="netstandard1.4">
         <dependency id="NETStandard.Library" version="1.6.1" />
+        <dependency id="System.Reflection.TypeExtensions" version="4.4.0" exclude="compile" />
```

Comment:
`exclude="compile"` means users' test projects won't reference assemblies in the `System.Reflection.TypeExtensions` package because of the NUnit package.